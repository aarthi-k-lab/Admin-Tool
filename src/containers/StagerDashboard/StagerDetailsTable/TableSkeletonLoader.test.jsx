import React from 'react';
import renderer from 'react-test-renderer';
import renderSkeletonLoader from './TableSkeletonLoader';

describe('<Table Skeleton Component />', () => {
  it('Skeleton renders correctly', () => {
    const tree = renderer
      .create(<renderSkeletonLoader />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
