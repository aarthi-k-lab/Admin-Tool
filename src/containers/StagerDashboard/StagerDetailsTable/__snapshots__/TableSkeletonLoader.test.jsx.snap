// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Table Skeleton Component /> Skeleton renders correctly 1`] = `<renderSkeletonLoader />`;
